pandas/core/missing.py

```diff
@@ -621,3 +622,24 @@ def fill_zeros(result, x, y, name, fill):
             result = result.reshape(shape)
 
     return result
+
+
+def maybe_fill(obj, value, errors):
```

Comment:
validation functions on the other hand will generally just return ``None`` OR raise. returning a boolean is a bit odd actually. You want simple as possible for things.